test/com/facebook/buck/swift/SwiftOSXBinaryIntegrationTest.java

```diff
@@ -121,4 +121,18 @@ public void swiftCallingObjCRunsAndPrintsMessageOnOSX() throws IOException {
         runResult.getStdout(),
         containsString("Hello ObjC\n"));
   }
+
+  @Test
```

Comment:
You should use `fbxctest` here.  See https://gist.github.com/ryu2/350a5d773e86fb70d5383cfd1e2c0ee0 for an example.

Unfortunately, after I made that change, it fails with this

```
    [junit] Testcase: swiftCallingComplexObjCRunsAndPrintsMessageOnOSX took 1.007 sec
    [junit] 	Caused an ERROR
    [junit] The same include path maps to multiple files:
    [junit]   Include path: SwiftCallsComplexObjC/FBASLParser.h
    [junit]   Conflicting files:
    [junit]     /private/var/folders/79/r8x23p3x5z962kr42g6qfnt0x713m8/T/junit-temp-path5100542076165353880/fbxctest/bin/FBControlCore.framework/Versions/A/Headers/FBASLParser.h
    [junit]     /private/var/folders/79/r8x23p3x5z962kr42g6qfnt0x713m8/T/junit-temp-path5100542076165353880/fbxctest/bin/FBControlCore.framework/Headers/FBASLParser.h
    [junit] com.facebook.buck.util.HumanReadableException: The same include path maps to multiple files:
    [junit]   Include path: SwiftCallsComplexObjC/FBASLParser.h
    [junit]   Conflicting files:
    [junit]     /private/var/folders/79/r8x23p3x5z962kr42g6qfnt0x713m8/T/junit-temp-path5100542076165353880/fbxctest/bin/FBControlCore.framework/Versions/A/Headers/FBASLParser.h
    [junit]     /private/var/folders/79/r8x23p3x5z962kr42g6qfnt0x713m8/T/junit-temp-path5100542076165353880/fbxctest/bin/FBControlCore.framework/Headers/FBASLParser.h
    [junit] 	at com.facebook.buck.apple.AppleDescriptions.convertToFlatCxxHeaders(AppleDescriptions.java:197)
    [junit] 	at com.facebook.buck.apple.AppleDescriptions.parseAppleHeadersForUseFromOtherTargets(AppleDescriptions.java:151)
    [junit] 	at com.facebook.buck.apple.AppleDescriptions.convertAppleHeadersToPublicCxxHeaders(AppleDescriptions.java:114)
    [junit] 	at com.facebook.buck.apple.AppleDescriptions.populateCxxConstructorArg(AppleDescriptions.java:224)
    [junit] 	at com.facebook.buck.apple.AppleDescriptions.populateCxxLibraryDescriptionArg(AppleDescriptions.java:288)
    [junit] 	at com.facebook.buck.apple.AppleLibraryDescription.requireSingleArchUnstrippedBuildRule(AppleLibraryDescription.java:352)
    [junit] 	at com.facebook.buck.apple.AppleLibraryDescription.requireUnstrippedBuildRule(AppleLibraryDescription.java:330)
    [junit] 	at com.facebook.buck.apple.AppleLibraryDescription.createLibraryBuildRule(AppleLibraryDescription.java:256)
    [junit] 	at com.facebook.buck.apple.AppleTestDescription.createTestLibraryRule(AppleTestDescription.java:366)
    [junit] 	at com.facebook.buck.apple.AppleTestDescription.createBuildRule(AppleTestDescription.java:225)
    [junit] 	at com.facebook.buck.apple.AppleTestDescription.createBuildRule(AppleTestDescription.java:1)
    [junit] 	at com.facebook.buck.rules.DefaultTargetNodeToBuildRuleTransformer.transform(DefaultTargetNodeToBuildRuleTransformer.java:48)
    [junit] 	at com.facebook.buck.rules.BuildRuleResolver.requireRule(BuildRuleResolver.java:120)
    [junit] 	at com.facebook.buck.rules.ActionGraphCache$1.visit(ActionGraphCache.java:152)
    [junit] 	at com.facebook.buck.rules.ActionGraphCache$1.visit(ActionGraphCache.java:1)
    [junit] 	at com.facebook.buck.graph.AbstractBottomUpTraversal.traverse(AbstractBottomUpTraversal.java:62)
    [junit] 	at com.facebook.buck.rules.ActionGraphCache.createActionGraph(ActionGraphCache.java:161)
    [junit] 	at com.facebook.buck.rules.ActionGraphCache.getActionGraph(ActionGraphCache.java:90)
    [junit] 	at com.facebook.buck.cli.TestCommand.runWithoutHelp(TestCommand.java:482)
    [junit] 	at com.facebook.buck.cli.AbstractCommand.run(AbstractCommand.java:198)
    [junit] 	at com.facebook.buck.cli.BuckCommand.run(BuckCommand.java:92)
    [junit] 	at com.facebook.buck.cli.Main.runMainWithExitCode(Main.java:1234)
    [junit] 	at com.facebook.buck.testutil.integration.ProjectWorkspace.runBuckCommandWithEnvironmentOverridesAndContext(ProjectWorkspace.java:478)
    [junit] 	at com.facebook.buck.testutil.integration.ProjectWorkspace.runBuckCommandWithEnvironmentOverridesAndContext(ProjectWorkspace.java:420)
    [junit] 	at com.facebook.buck.testutil.integration.TestDataHelper$CacheClearingProjectWorkspace.runBuckCommandWithEnvironmentOverridesAndContext(TestDataHelper.java:111)
    [junit] 	at com.facebook.buck.testutil.integration.ProjectWorkspace.runBuckCommand(ProjectWorkspace.java:354)
    [junit] 	at com.facebook.buck.swift.SwiftOSXBinaryIntegrationTest.swiftCallingComplexObjCRunsAndPrintsMessageOnOSX(SwiftOSXBinaryIntegrationTest.java:146)
```